accepted/0000-selective-versions-resolutions.md

```diff
@@ -0,0 +1,278 @@
+- Start Date: 2017-05-21
+- RFC PR: (leave this empty)
+- Yarn Issue: (leave this empty)
+
+# Summary
+
+Allow to select a nested dependency version via the `resolutions` field of
+the `package.json` file.
+
+ça veut dire qu'on doit pouvoir mettre des specs invalides (wrt vesion) dans
+lockfile.

```

Comment:
haha no, sorry, I took some notes while writing and forgot to remove them :P I'm going to amend the commit!